src/app/components/TransactionsTable/index.tsx

```diff
@@ -43,35 +46,60 @@ export default function TransactionsTable({ transactions }: Props) {
     <div className="shadow overflow-hidden border-b border-gray-200 rounded-lg">
       <div className="bg-white divide-y divide-gray-200">
         {transactions.map((tx) => (
-          <div key={tx.id} className="flex justify-between space-x-5 px-3 py-2">
-            <div className="overflow-hidden">
-              <div className="flex items-center">
-                <div className="flex-shrink-0 mr-3">
-                  {tx.type && renderIcon(tx.type)}
-                </div>
-                <div className="overflow-hidden">
-                  <div className="text-gray-900 truncate">{tx.title}</div>
-                  <div className="text-sm text-gray-500 truncate">
-                    {tx.subTitle}
+          <div className="px-3 py-2">
+            <Disclosure>
+              {({ open }) => (
+                <>
```

Comment:
The "Fragment" can be left out as `<div key {tx.id} />` is already the outer div.